sway/desktop/output.c

```diff
@@ -745,12 +745,18 @@ static void damage_child_views_iterator(struct sway_container *con,
 
 void output_damage_whole_container(struct sway_output *output,
 		struct sway_container *con) {
+	int shadow_sigma = con->shadow_enabled ? config->shadow_blur_sigma : 0;
+	int blur_size = con->blur_enabled ?
+			pow(2, config->blur_params.num_passes) * config->blur_params.radius : 0;
```

Comment:
Actually, I think this could probably be simplified to just `config->blur_params.radius : 0`?

Reply:
The blur needs more distance from its edges than the radius, this is just a nice way of calculating a good distance from the edges :)